crypto/tls/tls.go

```diff
@@ -69,6 +81,23 @@ func (cfg *ClientConfig) GetTLSConfig() (*tls.Config, error) {
 		config.Certificates = []tls.Certificate{clientCert}
 	}
 
+	if cfg.MinVersion != "" {
+		minVersion, ok := tlsVersions[cfg.MinVersion]
+		if !ok {
+			return nil, errors.New("failed to set minimum TLS version")
```

Comment:
I think it's helpful if the error includes the invalid TLS version (in case the value is non-obvious to the user):
```suggestion
			return nil, fmt.Errorf("failed to set minimum TLS version: %q", cfg.MinVersion)
```